x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/operator/exchange/ExchangeServiceTests.java

```diff
@@ -300,41 +301,15 @@ Set<Integer> runConcurrentTest(
         int numSources = randomIntBetween(1, 8);
         List<Driver> drivers = new ArrayList<>(numSinks + numSources);
         for (int i = 0; i < numSinks; i++) {
-            String description = "sink-" + i;
             ExchangeSinkOperator sinkOperator = new ExchangeSinkOperator(exchangeSink.get(), Function.identity());
             DriverContext dc = driverContext();
-            Driver d = new Driver(
-                "test-session:1",
-                "test",
-                0,
-                0,
-                dc,
-                () -> description,
-                seqNoGenerator.get(dc),
-                List.of(),
-                sinkOperator,
-                Driver.DEFAULT_STATUS_INTERVAL,
-                () -> {}
-            );
+            Driver d = TestDriverFactory.create(dc, seqNoGenerator.get(dc), List.of(), sinkOperator, () -> {});
```

Comment:
I think this can use the c'tor without the releasable:
```suggestion
            Driver d = TestDriverFactory.create(dc, seqNoGenerator.get(dc), List.of(), sinkOperator);
```